### PR TITLE
Replaced iconRight with iconPosition

### PR DESCRIPTION
**Pull request by hetanthakkar1**
Solved issue #2591

### CODE REVIEW

File: src/buttons/Button.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ export type ButtonProps = TouchableOpacityProps &
     containerStyle?: StyleProp<ViewStyle>;
     icon?: IconNode;
     iconContainerStyle?: StyleProp<ViewStyle>;
-    iconPosition?: string;
```
Comment (arpitBhalla) [inline src/buttons/Button.tsx]: don't remove this

Comment (hetanthakkar1) [reply src/buttons/Button.tsx]: okay!

Comment (hetanthakkar1) [reply src/buttons/Button.tsx]: why?

Comment (arpitBhalla) [reply src/buttons/Button.tsx]: it will break code

**Pull request closed without merging**